scss/_patterns_logo-section.scss

```diff
@@ -4,17 +4,16 @@
     @extend %small-caps-text;
   }
 
-  $logo-section-item-size: 8rem; // height of the logos on large screens
-  $logo-section-item-size-medium: 6rem; // height of the logos on medium screens
+  $logo-section-item-size: 6.5rem; // height of the logos on large screens
   $logo-section-item-size-small: 4rem; // height of the logos on small screens
 
   $logo-section-offset: 1rem; // offset by which rows are pulled closer together
-  $logo-section-offset-medium: 0.75rem; // height of the logos on medium screens
   $logo-section-offset-small: 0.5rem; // height of the logos on small screens
 
-  $logo-section-item-gap: 3rem;
-  $logo-section-item-gap-dense: calc(0.5 * $logo-section-item-gap);
+  $logo-section-item-gap: 2rem;
 
+  // TODO: Remove .p-logo-section--dense since it's been deprecated in favor
+  // of just using .p-logo-section instead
```

Comment:
Do we also need a deprecation notice in `releases.yml` and a notification in the docs like this?

![image](https://github.com/user-attachments/assets/d46fff4f-ebaf-4c1e-917a-3ac160649884)

Reply:
Yes, would be good for visibility in the docs.